src/contributor.ts

```diff
@@ -34,6 +31,14 @@ const parseContributors = (salt: string, content: Content): Contributor[] =>
         image: fromNullable(contributor.bylineLargeImageUrl).fmap(url => ({
             srcset: contributorSrcset(url, salt, Dpr.One),
             src: src(salt, url, 64, Dpr.One),
+            dpr2Srcset: contributorSrcset(url, salt, Dpr.Two),
```

Comment:
We might want to offer a couple of higher res widths on line 21 in this file. The cutout is somewhat larger than the avatar and so could probably benefit on DPR 2 screens? Maybe `400` and `600`?